### DISCUSSION TITLE
Filter buttons at top of meeting list don't activate on hover

### DISCUSSION THREAD

**Discussion by BDS419** [code4recovery/12-step-meeting-list#635] [Q&A]
The wordpress theme seems to be covering up the meeting filter buttons, Everywhere, Day Time etc. Our website is lewiscountyaa.org. We are a rural county with little to no tech expertise. Hope you can help. Thanks.

Brian S.
Web Servant, District 28, Washington State

**Comment by tech2serve** [comment]
Hey Brian (@BDS419),

Best I can tell, we completely missed your question here, and I feel horrible about that. I apologize, and wish there was a way to roll back the clock. 

Looking at your website, it appears you got it working, yes? I would be curious to hear what you discovered and how you fixed it. It might help someone else in the future.

Tim
